client/state/login/reducer.js

```diff
@@ -124,6 +125,22 @@ export const twoFactorAuthPushPoll = createReducer( { inProgress: false, success
 	[ TWO_FACTOR_AUTHENTICATION_PUSH_POLL_COMPLETED ]: state => ( { ...state, inProgress: false, success: true } ),
 } );
 
+export const socialAccount = createReducer( { isCreating: false }, {
+	[ SOCIAL_CREATE_ACCOUNT_REQUEST ]: () => ( { isCreating: true } ),
+	[ SOCIAL_CREATE_ACCOUNT_REQUEST_FAILURE ]: ( state, { error } ) => ( { isCreating: false, createError: error } ),
+	[ SOCIAL_CREATE_ACCOUNT_REQUEST_SUCCESS ]: ( state, { data: { username, bearerToken } } ) => ( {
+		isCreating: false,
+		username,
+		bearerToken
+	} ),
+	[ SOCIAL_LOGIN_REQUEST_FAILURE ]: ( state, { error } ) => ( {
+		...state,
+		requestError: error
+	} ),
+	[ USER_RECEIVE ]: state => ( { ...state, bearerToken: null, username: null } ),
```

Comment:
I was wondering if we should set state to its initial value:
```js
{ isCreating: false }
```
for `USER_RECEIVE` and `LOGIN_REQUEST` action types.

I wanted to double check to be on a safe side.